### DISCUSSION TITLE
[灌水]有人知道这是哪个乐队吗？

### DISCUSSION THREAD

**Discussion by WORLD-executeME** [share121/inter-knot#1170] [其他]
![屏幕截图 2024-08-10 191336](https://github.com/user-attachments/assets/1dc9ebfb-a6e0-4e5a-93ae-c95b96a09d37)

前几天和朋友去看了场LIVE,中途有一个让我很在意的乐队，第一首歌时主唱先是声音很小，又突然像是被激励了一样越唱越好，有点拼命的感觉，第二首曲子的时候还有个姑娘边哭边跑出live house了，应该是太感动了吧，有些在意这个乐队，但没注意乐队名，图是LIVE时拍的，可能有些模糊，有认识的人可以告诉我乐队名吗？

**Comment by zyyyrsama** [comment]
还在go，还在go

**Comment by hzjwork** [comment]
我记得乐队名好像叫做我去，主唱像小动物一样很可爱，就是太拼命了

**Reply by WORLD-executeME** [reply]
主唱太拼命了

**Comment by llxcx4** [comment]
主唱太拼命了

**Reply by WORLD-executeME** [reply]
我要拉黑你！

**Comment by Kamisato100** [comment]
好像邦什么疑似卖沟的，真是奇怪的名字

**Comment by gaojiaobei** [comment]
我当时在后台那边看到这个乐队了，当时四个人在那里庆祝，那个棕色头发的女孩在后面突然喊着什么“为什么要演奏春日影”，然后就跑走了，吓了我一跳。
这就是年轻人的爱与热情吧，真好啊。

**Reply by WORLD-executeME** [reply]
是啊

**Comment by bananice12138** [comment]
短发的主唱有点眼熟，以前好像看过她主唱的live，我记得乐队是叫……crychic？

**Reply by WORLD-executeME** [reply]
谢谢，我去搜一下她们的账号
